<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>The Backrooms</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/Arrows.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="escapeBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>31</width>
      <height>291</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dieBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>404</y>
      <width>46</width>
      <height>47</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="arrowlb">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>411</y>
      <width>22</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/image 24.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>escapeBtn</zorder>
   <zorder>dieBtn</zorder>
   <zorder>arrowlb</zorder>
   <zorder>backBtn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
